code/go/0chain.net/smartcontract/storagesc/handler.go

```diff
@@ -142,19 +142,43 @@ func (srh *StorageRestHandler) getAverageWritePrice(w http.ResponseWriter, r *ht
 //      required: true
 //      in: query
 //      type: string
+//    + name: offset
+//      description: offset
+//      in: query
+//      type: string
+//    + name: limit
+//      description: limit
+//      in: query
+//      type: string
+//    + name: sort
+//      description: desc or asc
+//      in: query
+//      type: string
 //
 // responses:
 //  200: stringArray
 //  400:
 func (srh *StorageRestHandler) getBlobberIdsByUrls(w http.ResponseWriter, r *http.Request) {
-	urlsStr := r.URL.Query().Get("blobber_urls")
+	var (
+		urlsStr            = r.URL.Query().Get("blobber_urls")
+		offsetString       = r.URL.Query().Get("offset")
+		limitString        = r.URL.Query().Get("limit")
+		isDescendingString = r.URL.Query().Get("is_descending")
```

Comment:
We can have a function to get and parse the `offset`, `limit`, `sort` from query string to avoid the repeating work in all api handlers.

Also, please use `sort` as the param name instead of `is_descending`, it is a bit long for typing, and we should align it with the swagger doc comments declared above.

Reply:
is_descending has more sense than sort I think, it is bool value that tells should set be sorted ascending or descending, if we name it sort, than it won't be obvious what is meant here

Reply:
moved parameters to function

Reply:
but we are actually passing string value of 'desc' and 'asc'. So the query will be `?is_descending=desc` or `?is_descending=asc`. Please check the param parsing code.

Reply:
ah my bad, you are right, I'll fix it